pages/api/cloudinary/deleteImage.js

```diff
@@ -0,0 +1,25 @@
+import cloudinary from "cloudinary";
+
+export default async function handler(request, response) {
+  if (request.method !== "DELETE") {
+    return response.status(405).json({ message: "Method Not Allowed" });
```

Comment:
```suggestion
    response.status(405).json({ message: "Method Not Allowed" });
    return
```
Please split this into two lines, since it would suggest that the response object is returned, which should not be the case. calling response.json() itself triggers the response, not returning it.